scripts/lib/CIME/case_st_archive.py

```diff
@@ -143,11 +143,13 @@ def _datetime_str_mpas(date):
     '0011-12-09_00435'
     """
 
-    format_string = "{year:04d}-{month:02d}-{day:02d}_{seconds:05d}"
+    format_string = "{year:04d}-{month:02d}-{day:02d}_{hours:02d}:{minutes:02d}:{seconds:02d}"
```

Comment:
@mfdeakin-sandia ?

Reply:
Yes, this change is correct for MPAS rpointer files